### ISSUE TITLE
Question about trainning

### ISSUE THREAD

**Issue by HeathHose** [Francis-Rings/StableAnimator#63]
Thanks for the impressive work.  I used my own data for training, and as the number of training steps increased, the intermediate results became worse and worse until they completely collapsed. I suspected that the quality of the dataset was poor, so I replaced it with ubc-fasion, a public dataset, but the results remained the same.

Have you encountered similar issues during the training process, and do you have any insights?  Looking forward to your reply~

**Comment by HeathHose** [comment]
This is the visualization after 500 steps
![combined_frames](https://github.com/user-attachments/assets/da387526-1c09-44b3-9c97-e8ffee451354)

**Comment by HeathHose** [comment]
This is the visualization after 10000 steps
![combined_frames_20241225-101210](https://github.com/user-attachments/assets/21501b8c-480b-47b0-aeb4-d3ba41497f0f)

**Comment by HeathHose** [comment]
I initialize the environment according to requirements.txt in cuda12.4+ubuntu22.4

**Comment by Francis-Rings** [comment]
Hi, I haven't encountered this issue before. Based on your results, here are three suggestions to ensure you're training StableAnimator correctly: (1) Check the dimensions of your training and validation videos. It seems that the human skeleton images are misaligned with the reference image. (2) Ensure that the pose is being fed into the PoseNet correctly. Your results don't follow the motion pattern of the driving video, which is quite weird. (3) Confirm that the trainable parameters are being updated during training and that the loss values are within a normal range. You can firstly initialize StableAnimator with SVD weights to see if the issue occurs.
